### DISCUSSION TITLE
Angular 13 tailwind less-loader module build failed

### DISCUSSION THREAD

**Discussion by rockcoolt** [tailwindlabs/tailwindcss#7348] [Help]
Hello i need your help!

**What version of Tailwind CSS are you using?**

3.0.18

**What build tool (or framework if it abstracts the build tool) are you using?**

Angular Webpack

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Describe your issue**

In my less file i have this css description:

```
& .list {
     @apply w-full h-[calc(100vh_-_10rem)] overflow-y-auto px-4;
}
```

When the application build i get an Error:
`@apply rule is missing block or ending semi-colon`

![image](https://user-images.githubusercontent.com/18094312/152740073-8889abae-1843-4a86-9cd0-5b04ff93729c.png)

I don't understand what is the problem :(
Can you help me pls ?

**Comment by EickeOe** [comment]
Did you solve this problem please?
